libopenage/gamestate/game_main.cpp

```diff
@@ -22,28 +22,32 @@ GameMain::GameMain(const Generator &generator)
 	// players
 	unsigned int i = 0;
 	for (auto &name : generator.player_names()) {
-		this->players.emplace_back(this->add_civ(i), i, name);
+		this->players.push_back(new Player(this->add_civ(i), i, name));
```

Comment:
References > pointers. I don't see any particular reason to need pointers here - surely the number of players doesn't change after initialisation? Shouldn't be any issue with invalidated iterators